src/STM32RTC.h

```diff
@@ -34,6 +34,9 @@
   ******************************************************************************
   */
 
+#if defined(__cplusplus) && !defined(C_IMPLEMENTATION)
```

Comment:
I don't think it is needed to add an extra definition
```suggestion
#if defined(__cplusplus)
```